konflux-pr-check.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -ex
+
+echo "INSTALL DEPENDENCIES"
+microdnf install -y python39
+pip3 install --no-cache-dir -r requirements.txt
+pip3 install --upgrade pip
```

Comment:
not big deal, I think we should first upgrade pip and then use it to install dependencies